pages/firebase-firsttry.js

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from 'react';
+import { collection, onSnapshot } from 'firebase/firestore';
+import Link from 'next/link';
+
+import { db } from '../firebase/initFirebase';
+
+const FirebaseFirst = () => {
+  const [articleList, setArticleList] = useState([]);
+  const articlesCollectionRef = collection(db, 'articles');
+
+  useEffect(() => {
+    onSnapshot(articlesCollectionRef, (snapshot) => {
```

Comment:
I would recommend you doing the same abstraction as my project had - that `api` folder.